code/modules/vehicles/vehicle.dm

```diff
@@ -94,12 +94,14 @@
 /obj/vehicle/relaymove(mob/user, direction)
 	if(user.incapacitated())
 		unbuckle_mob(user)
+		return
 
+	if(world.time < next_vehicle_move)
+		return
+	next_vehicle_move = world.time + vehicle_move_delay
```

Comment:
Moved the delay check here, so we don't get spammed with the failure message for lack of key as often.